### DISCUSSION TITLE
@formily/antd 中的Input组件默认值存在问题

### DISCUSSION THREAD

**Discussion by Chen-Weidong** [alibaba/formily#2813] [Q&A 2.x] [answered]
@formily/antd 中的Input组件设置默认值后，提交时报错“该字段是必填字段”
Sandbox：[https://codesandbox.io/s/sad-meitner-15r3p?file=/App.tsx](https://codesandbox.io/s/sad-meitner-15r3p?file=/App.tsx)
![image](https://user-images.githubusercontent.com/73837770/153538351-bf8f947c-e192-4ad8-bed3-2a62882b0019.png)

另外，有没有办法在 “x-reactions” 中设置 @formily/antd 的enum属性呢，没找到 setEnum这个方法。

**Comment by janryWang** [comment, marked as answer]
用default属性，defaultValue formily是收集不到的
